ensemble/ctf/manager.py

```diff
@@ -12,25 +14,27 @@
 
 
 def _name_encode(name):
-    return urlsafe_b64encode(name.encode('utf-8'))
+    # Accepts a unicode input and returns a unicode output
+    return urlsafe_b64encode(name.encode('utf-8')).decode('utf-8')
```

Comment:
This should maybe not return a unicode string... It's being used for a file path, so I'm actually not sure...